### PR TITLE
add short names for gitsource and gitsourceanalysis

### PR DESCRIPTION
**Pull request by corinnekrych**
@MatousJobanek so we can have sth shorter like:
```
➜  oc get all,dc,svc,dc,bc,route,cp,src,srcan
NAME                READY     STATUS      RESTARTS   AGE
pod/myapp-1-6lgmq   1/1       Running     0          5m
pod/myapp-1-build   0/1       Completed   0          6m

NAME                            DESIRED   CURRENT   READY     AGE
replicationcontroller/myapp-1   1         1         1         5m

NAME            TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
service/myapp   ClusterIP   172.30.220.21   <none>        8080/TCP   6m

NAME                                       REVISION   DESIRED   CURRENT   TRIGGERED BY
deploymentconfig.apps.openshift.io/myapp   1          1         1         config,image(myapp:latest)

NAME                                   TYPE      FROM         LATEST
buildconfig.build.openshift.io/myapp   Source    Git@master   1

NAME                               TYPE      FROM          STATUS     STARTED         DURATION
build.build.openshift.io/myapp-1   Source    Git@e59fe75   Complete   6 minutes ago   1m6s

NAME                                   DOCKER REPO                        TAGS      UPDATED
imagestream.image.openshift.io/myapp   172.30.1.1:5000/local-test/myapp   latest    5 minutes ago

NAME                             HOST/PORT                                PATH      SERVICES   PORT      TERMINATION   WILDCARD
route.route.openshift.io/myapp   myapp-local-test.192.168.99.100.nip.io             myapp      8080                    None

NAME                                      STATUS
component.devconsole.openshift.io/myapp   Deployed

NAME                                                  STATUS
gitsource.devconsole.openshift.io/example-gitsource

NAME                                                                   STATUS
gitsourceanalysis.devconsole.openshift.io/example-gitsource-analysis   true
```

### CODE REVIEW

File: deploy/crds/devconsole_v1alpha1_gitsource_crd.yaml [hunk 1]
```diff
@@ -9,6 +9,8 @@ spec:
     listKind: GitSourceList
     plural: gitsources
     singular: gitsource
+    shortNames:
+      - src
```
Comment (sbose78) [inline deploy/crds/devconsole_v1alpha1_gitsource_crd.yaml]: `src` is too overloaded a term. How about gitsrc or gs ( along the lines of `bc` , `dc`)?

File: deploy/crds/devconsole_v1alpha1_gitsourceanalysis_crd.yaml [hunk 2]
```diff
@@ -80,6 +82,10 @@ spec:
                 a reason for the build type detection failure.
                 Possible values are [NotSupportedType, DetectionFailed, InternalFailure]
               type: string
+  additionalPrinterColumns:
```
Comment (sbose78) [inline deploy/crds/devconsole_v1alpha1_gitsourceanalysis_crd.yaml]: Nice!

File: deploy/crds/devconsole_v1alpha1_gitsource_crd.yaml (hunk 1 above)
Comment (tinakurian) [reply deploy/crds/devconsole_v1alpha1_gitsource_crd.yaml]: I agree

File: deploy/crds/devconsole_v1alpha1_gitsourceanalysis_crd.yaml [hunk 3]
```diff
@@ -9,6 +9,8 @@ spec:
     listKind: GitSourceAnalysisList
     plural: gitsourceanalyses
     singular: gitsourceanalysis
+    shortNames:
+      - srcan
```
Comment (tinakurian) [inline deploy/crds/devconsole_v1alpha1_gitsourceanalysis_crd.yaml]: If the other shortname is "gs" maybe this can be "gsa" or "gsan" or something?

Comment (pratikjagrut) [reply deploy/crds/devconsole_v1alpha1_gitsourceanalysis_crd.yaml]: "gs" and "gsa" sounds good.

Comment (corinnekrych) [reply deploy/crds/devconsole_v1alpha1_gitsourceanalysis_crd.yaml]: 👍 
done with 29fe71b

**Pull request merged**